### PR TITLE
Added new tool: obfs4proxy

### PR DESCRIPTION
**Pull request by mrsmith0x00**
Package `obfsproxy` isn't in a Arch repository any more.

### CODE REVIEW

File: packages/obfs4proxy/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+pkgname=obfs4proxy
+pkgver=0.0.7
+pkgrel=1
+pkgdesc='A pluggable transport proxy written in Go'
+groups=('blackarch' 'blackarch-proxy' 'blackarch-networking')
+arch=('i686' 'x86_64')
+url='https://gitweb.torproject.org/pluggable-transports/obfs4.git'
+license=('BSD')
+makedepends=('git' 'go')
```
Comment (FFY00) [inline packages/obfs4proxy/PKGBUILD]: Should be `go-pie`, not `go`.

File: packages/obfs4proxy/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+optdepends=('tor')
+source=("git+https://git.torproject.org/pluggable-transports/obfs4.git#tag=obfs4proxy-${pkgver}")
+sha1sums=('SKIP')
+
+build() {
+  cd "$srcdir/obfs4/obfs4proxy"
+  
+  GOPATH="$srcdir" GOBIN="$PWD" go get
```
Comment (FFY00) [inline packages/obfs4proxy/PKGBUILD]: Use go build.

https://wiki.archlinux.org/index.php/Go_package_guidelines#Sample_PKGBUILD_for_when_only_a_single_source_file_is_available

**Comment by noptrix** [comment]
@mrsmith0x00 any updates on this? :)

**Comment by mrsmith0x00** [comment]
@noptrix I will check and I will prepare during week-end, yet I can't. ;)

**Comment by noptrix** [comment]
thx @mrsmith0x00

File: packages/obfs4proxy/PKGBUILD (hunk 1 above)
Comment (mrsmith0x00) [reply packages/obfs4proxy/PKGBUILD]: Complete.

File: packages/obfs4proxy/PKGBUILD (hunk 2 above)
Comment (mrsmith0x00) [reply packages/obfs4proxy/PKGBUILD]: Complete.

**Comment by mrsmith0x00** [comment]
@noptrix sorry, what couldn't make it quickly.

**Comment by noptrix** [comment]
@mrsmith0x00 absolutely no problem. thanks a lot. @FFY00 can we merge in as the test were passed?

**Pull request merged**